### DISCUSSION TITLE
The login page cannot log in normally

### DISCUSSION THREAD

**Discussion by Teklvtn** [apache/airflow#26948] [General]
### Apache Airflow version

2.4.1

### What happened

The page keeps saying the password is wrong
I created several new users, all of them are Admin roles, but I can't log in normally, and I can't log in using the admin password of airflow's standalone_admin_password.txt

### What you think should happen instead

There may be a bug in the authentication module

### How to reproduce

Install airflow using pip, which should be reproducible through airflow standalone

### Operating System

ubuntu

### Versions of Apache Airflow Providers

2.4.1

### Deployment

Other

### Deployment details

via pip or wheel

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by Teklvtn** [comment]
I successfully logged in, users can be created normally through the users create command, I just forgot to enter the --password command, the shell prompted me to enter and repeat the password, then it works! ! But I still think it's a bug

**Comment by potiuk** [comment]
> I successfully logged in, users can be created normally through the users create command, I just forgot to enter the --password command, the shell prompted me to enter and repeat the password, then it works! ! But I still think it's a bug

What is a bug exactly? It is not clear what is the bug from this report - since you were able to login. Can you please explain what the bug is ? converting it into discussion until you do.

**Comment by NargesGoetzsche** [comment]
I have same issue with 2.7.2 and followed above instruction but with no luck.
I have wiped DB and re-created accounts but still can not login. any idea?

**Reply by potiuk** [reply]
Nope
